forms/models/behaviour.js

```diff
@@ -23,6 +23,13 @@ define(function (require) {
   var Behaviour;
 
   Expression.fn['formelement.value'] = function (name) {
+    var el = this.getElement(name);
+    if (!el) {
+      /* eslint-disable no-console */
+      console && console.warn('Conditional Logic uses a hidden Element: ' + name);
+      /* eslint-enable no-console */
+      return undefined;
+    }
     return this.getElement(name).val();
```

Comment:
why do we need to do this.getElement(name) (at line 33) as we already have el from above ?